Src/Web/Web.Nuget/Package.nuspec

```diff
@@ -25,7 +25,7 @@
         <dependency id="Microsoft.ApplicationInsights" version="[$coresdkversion$]" />
         <dependency id="Microsoft.ApplicationInsights.WindowsServer" version="$version$" />
         <dependency id="Microsoft.AspNet.TelemetryCorrelation" version="1.0.0-preview1-10407-02" />
```

Comment:
@lmolkova NuGet hosts the version `1.0.0-preview1-10414-01` of `TelemetryCorrelation`. Should we update the nuspec or it works all right?